src/services/dashboard.js

```diff
@@ -38,41 +41,46 @@ export async function queryControllers(params) {
   try {
     const { datastoreConfig, selectedDateRange } = params;
 
-    const endpoint = `${datastoreConfig.elasticsearch}/${getAllMonthsWithinRange(
+    const indices = `${getAllMonthsWithinRange(
       datastoreConfig,
       datastoreConfig.run_index,
       selectedDateRange
     )}/_search`;
 
+    const endpoint = `${datastoreConfig.pbench_server}/elasticsearch`;
+
     return request.post(endpoint, {
-      params: {
-        ignore_unavailable: true,
-      },
       data: {
-        filter: {
-          range: {
-            '@timestamp': {
-              gte: selectedDateRange.start,
-              lte: selectedDateRange.end,
-            },
-          },
+        indices,
```

Comment:
This is more a comment on the original Pbench server API design, but it didn't really start bugging me until I saw it appear repeatedly in the dashboard changes.

This is not well-formed JSON, because we're just throwing in an unlabeled string. This ought to be another key, like "uri" ... although `uri` is misleading since it's really only the resource part of the uri. So I suppose `"resource"` or `"target_resource"` would be better.

I'm not inclined to push this issue too far, since these changes are short lived, and when we have a real API we'll be cutting off the pass-through APIs anyway. Still, it's going to continue to bug me every time I see it.

Reply:
I was trying to make eslint happy with https://eslint.org/docs/rules/object-shorthand. Apparently in ES6 `var foo = {x, y, z};` = `var foo = {x:x, y:y, z:z};`

Reply:
Ah; and I knew that, too, but I clearly hadn't completely twisted my brain from Python to Javascript! Sorry 'bout that.

On the other hand, a JSON key of "indices" isn't really accurate, either, but that's not a big deal in this context.